### PR TITLE
Feat: Authentication is added

### PR DESCRIPTION
**Pull request by ramalingaa**
User can now Login/Signup

### CODE REVIEW

File: src/frontend/components/home/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Home = () => {
```
Comment (Parul1999) [inline src/frontend/components/home/Home.jsx:3]: If not needed , the empty component can be deleted.

File: src/frontend/components/authentication/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    <div className = "login-landing-wrapper">  
+        <div className = "login-title-wrapper">
+            <h1 className = "login-logo-title">One Mile</h1>
+            <p className = "title-subtext">One Mile  helps you connect and share with the people in your life.</p>
+        </div>
+        <div className = "login-page-wrapper">
+            <div className = "login-card-wrapper">
+                <p className = "text-large login-header">Login</p>
+                <label className = "input-label">
+                    <input type = "email" placeholder = " " name = "email"className = "i-text input-name login-input" onChange = {updateUserData}/>
```
Comment (Parul1999) [inline src/frontend/components/authentication/Login.jsx:76]: Placeholder is empty, if not needed can be removed

**Review by Parul1999 — COMMENTED** [review]
-- good code-structure 
-- Remove unused components/ attributes

File: src/frontend/components/home/Home.jsx (hunk 1 above)
Comment (ramalingaa) [reply src/frontend/components/home/Home.jsx:3]: The component will be added later, created it for redirecting after login

File: src/frontend/components/authentication/Login.jsx (hunk 2 above)
Comment (ramalingaa) [reply src/frontend/components/authentication/Login.jsx:76]: Placeholder is needed make the label animation. It is part of the css

**Pull request merged**